subsys/bluetooth/audio/pacs_internal.h

```diff
@@ -22,22 +22,20 @@ struct bt_pac_codec {
 #define BT_CODEC_CAP_DRM		0x0a
 #define BT_CODEC_CAP_DRM_VALUE		0x0b
 
-struct bt_pac_codec_capability {
-	uint8_t  len;			/* Codec Capability length */
-	uint8_t  type;			/* Codec Capability type */
-	uint8_t  data[0];		/* Codec Capability data */
+struct bt_pac_ltv {
+	uint8_t  len;
+	uint8_t  type;
+	uint8_t  value[0];
 } __packed;
 
-struct bt_pac_meta {
-	uint8_t  len;			/* Metadata Length */
-	uint8_t  value[0];		/* Metadata Value */
+struct bt_pac_ltv_hdr {
+	uint8_t  len;
+	uint8_t  data[0];
```

Comment:
Is there `data` field ever used?

In contrast, it does seem a bit odd to have `data` in a `hdr`

Suggest to remove (and then maybe the header struct is a bit overkill to have).

Reply:
It's information header for the data behind. It is a common pattern to define network packets. Even if `data[0]` is not explicitly referenced it still gives the information that there is (or not if `len` == 0) data behind.

Reply:
> It is a common pattern to define network packets like this.

Might be, but if we compare it to other `hdr` structs in Zephyr it is very uncommon. In fact, we have 53 structs that define a header struct (based on the `_hdr` suffix), and only 5 of them have a `data` field. 

I am OK with leaving it, but would still suggest to remove (especially since it is also not used).

Reply:
Ha! Emil, thanks to you, I realized I introduced a bug!

Reply:
> Ha! Emil, thanks to you, I realized I introduced a bug!

I didn't catch that when reviewing :D Is that bug something we can easily add some testing for in the unicast BSIM tests?

Reply:
It's the matter of size of `struct bt_pac` PAC Record that's minimum size is 7 according to spec.
```
struct bt_pac {
	struct bt_pac_codec codec;	/* Codec ID */
	struct bt_pac_ltv_data cc;	/* Codec Specific Capabilities */
	struct bt_pac_ltv_data meta;	/* Metadata */
} __packed;
```

Thus the `sizeof(struct bt_pac)` has to be 7 bytes :)